docs/devices/index.rst

```diff
@@ -99,3 +99,5 @@ Device APIs
    :members:
 .. automodule:: hwilib.devices.coldcard
    :members:
+.. automodule:: hwilib.devices.jade
+  :members:
```

Comment:
I think the indentation might be wrong here as it is not showing up in the docs preview.

Reply:
Apologies - I hope it's now repaired.

Reply:
Hmm. It isn't showing up in the preview, but it does show up locally. So I guess it's fine.